index.html

```diff
@@ -115,7 +115,7 @@
     *tar.xz)
       ${get} "${url}" | tar xfJ - || return 1
       # shellcheck disable=SC2086
-      install_subctl subctl*/subctl*${os}-${architecture}*
+      install_subctl subctl*/subctl
```

Comment:
Since get.submariner.io can install older releases, this needs to handle both naming schemes.